webapp/platform/components/src/compass-ui/utils/color-utils.ts

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {decomposeColor, hslToRgb} from '@mui/material';
+import {normal} from 'color-blend';
```

Comment:
Could you add this library to the dependencies of this package? We're currently getting it from the web app importing `compass-components`, so it'd be better to be explicit about it as a dependency